### PR TITLE
Feature/zenko 3513 bump pensieve api

### PR DESCRIPTION
**Pull request by rachedbenmustapha**
Updates pensieve-api and zenko-operator to fix erroneous attempts at sending metrics to CloudWatch.

### CODE REVIEW

File: solution/deps.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jabba:
 pensieve-api:
   sourceRegistry: registry.scality.com/pensieve-api
   image: pensieve-api
-  tag: 1.0.0-alpha.8
+  tag: 1.0.0-rc.2
```
Comment (jonathan-gramain) [inline solution/deps.yaml:42]: just checking, is this one still tagged `rc.2`?

**Review by jonathan-gramain — APPROVED** [review]

**Review by miniscruff — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]

File: solution/deps.yaml (hunk 1 above)
Comment (benmur) [reply solution/deps.yaml:42]: Yes, it’s its own rc number independent of where it’s used. Could be declared GA soon :)

Comment (rachedbenmustapha) [reply solution/deps.yaml:42]: Yes, it’s its own rc number, independent of where it’s used. Could be declared GA soon :)

**Comment by rachedbenmustapha** [comment]
Ping

**Pull request merged**